<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SitesView</class>
 <widget class="QDockWidget" name="SitesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Список источников</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="textGeo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Название сайта</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textURL"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Формат</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shapefile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jpg</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название местности</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnNew">
        <property name="text">
         <string>Новая запись</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
